docs/directory-structure.md

```diff
@@ -0,0 +1,212 @@
+All directories in this file are relative to project root
+
+Each directory should start with three hashes 
+
+###.github
+configuration files that interact with GitHub services 
+###.github/workflows
+workflow (interchangeably, actions) that runs by GitHub Actions service when code pushes to GitHub.
+###CreeDictionary
+
+A Django site with _django apps_ inside
+
+###CreeDictionary/API
+One of the django apps of our project. It defines click-in-text (our other project) API. 
+
+###CreeDictionary/API/migrations
+Django Migrations of the API app
+
+Migrations are not used to migrate database on production
+The only times migrations are needed are when we re-build the database and
+create empty tables with `manage.py migrate`
+
+This includes when we manually build test database, 
+and when our unit tests build test in memory database. 
+
+0000_initial.py migration is re-generated based on models.py everytime when the database is rebuilt.  
+
+This directory is managed by the rebuild script and does not require manual intervention
```

Comment:
yoinked from that local README.md. Four local REAMDE markdowns were deleted/edited. If the original markdown has a lot of details I kept them and referred to them here.